tests/emit.spec.ts

```diff
@@ -93,7 +93,7 @@ describe('emitted', () => {
       name: 'Parent',
       setup(props, { emit }) {
         return () =>
-          h(Child, { onHello: (...events) => emit('parent', ...events) })
+          h(Child, { onHello: (...events: any[]) => emit('parent', ...events) })
```

Comment:
I think unknown is more correct here (seems that is how it is typed in vue/core)